clang/test/3C/supernested.c

```diff
@@ -39,7 +39,7 @@ int vsf_sysdep_has_capabilities_as_non_root(void) {
 static void do_sanity_checks(void) {
   //CHECK: static void do_sanity_checks(void) _Checked {
   {
-    //CHECK: {
+    //CHECK: _Unchecked {
```

Comment:
Is this `_Unchecked` because of an unsafe external call, e.g., `die` ? Just curious if you know the reason.

Reply:
Because of a call to a function with unchecked types (and itypes) is incorrectly inferred as being unchecked, as above

Reply:
Fixed in latest commit